packages/components/src/theme/components/button.modifier.ts

```diff
@@ -1,28 +1,34 @@
 import { Components, Theme } from '@mui/material';
-import { CARIBBEAN_GREEN, KEPPEL, WHITE, buttonDisabled } from '../colors';
+import { CARIBBEAN_GREEN, WHITE, buttonDisabled } from '../colors';
 
 export const MuiButton: Components<Theme>['MuiButton'] = {
   styleOverrides: {
-    root: {
-      borderRadius: '3px',
-      textTransform: 'none',
-      '&.Mui-disabled': {
-        backgroundColor: buttonDisabled.main,
-        color: WHITE
-      }
-    },
-    contained: {
-      backgroundColor: KEPPEL,
-      '&:hover': {
-        backgroundColor: CARIBBEAN_GREEN,
-        color: WHITE
-      }
-    },
-    outlined: {
-      '&:hover': {
-        backgroundColor: CARIBBEAN_GREEN,
-        color: WHITE
-      }
+    root: ({ theme }) => {
+      const {
+        palette: {
+          secondary: { main }
```

Comment:
@nebula-aac require your review here while defining secondary.
In `paletteOptions interface` primary was only defined , so do we required to define secondary there too? Or it will work?

Reply:
And defining like this will also take care for ui which have only light mode like cloud? @nebula-aac